website/src/components/Footer/index.tsx

```diff
@@ -17,26 +17,68 @@ import {
 
 export default function Footer() {
   return (
-    <footer className="pt-12 relative bg-neutral-900 text-neutral-500">
+    <footer className="relative bg-neutral-900 text-neutral-500">
+      <div className="flex flex-col py-20 justify-center items-center text-center">
+        <h3 className="text-neutral-200 mb-4 text-4xl md:text-6xl text-pretty tracking-tight justify-center font-semibold">
+          Ready to get started?
+        </h3>
+        <h4 className="text-neutral-200 text-md tracking-tight">
+          Give your organization the protection it deserves.
+        </h4>
+        <div className="flex justify-center items-end w-fit gap-4 mt-8">
+          <div className="mx-auto">
+            <button
+              type="button"
+              className="text-sm w-48 inline-flex shadow-lg justify-center items-center py-2 px-5 font-semibold text-center text-white rounded bg-primary-450 hover:ring-2 hover:ring-primary-500 duration-50 transform transition"
+            >
+              <Link href="/contact/sales">Book a demo</Link>
+              <HiArrowLongRight className="ml-2 -mr-1 w-6 h-6" />
+            </button>
+            <p className="mt-3 text-xs text-neutral-400">
+              Get a personalized walkthrough of Firezone.
+            </p>
+          </div>
+          <div className="mx-auto">
+            <button
+              type="button"
+              className="text-sm w-48 inline-flex justify-center items-center py-2 px-5 font-semibold text-center text-white rounded hover:ring-2 hover:ring-primary-400 duration-50 transform transition"
```

Comment:
Just missing the `group` class here to trigger the hover effects on mouseover of the button.

```suggestion
              className="group text-sm w-48 inline-flex justify-center items-center py-2 px-5 font-semibold text-center text-white rounded hover:ring-2 hover:ring-primary-400 duration-50 transform transition"
```